code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -170,6 +176,13 @@
 			playsound(loc, 'sound/effects/Kidanclack2.ogg', 50, 0) //Credit to DrMinky (freesound.org) for the sound.
 			m_type = 2
 
+		if("creaks")
+			var/M = handle_emote_param(param)
+
+			message = "<B>[src]</B> creaks[M ? " at [M]" : ""]."
+			playsound(loc, 'sound/voice/dionatalk1.ogg', 50, 0)
```

Comment:
If was taken from somewhere that required any attribution, we need to state it in a comment. Please link to the licence where the sound is from?